### PR TITLE
Update information about assets

### LINKED ISSUE #3227: Update assets contributions in the docs
The section for how to add assets is outdated. We should change it for how we do it now

https://rotki.readthedocs.io/en/latest/contribute.html#adding-new-assets-to-rotki

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3227

### CODE REVIEW

**Comment by yabirgb** [comment]
I linked the information in the assets repo and didn't move it to the docs because soon this will change with the new globaldb schema

File: docs/contribute.rst [hunk 1]
```diff
@@ -72,44 +72,31 @@ Our releases work like this:
 Adding new assets to rotki
 ============================
 
-To add new assets to rotki you need to edit `all assets file <https://github.com/rotki/rotki/blob/239552b843cd8ad99d02855ff95393d6032dbc57/rotkehlchen/data/all_assets.json>`__.
+To add new assets for rotki you will have to edit `the SQL file in the last update at the assets repository <https://github.com/rotki/assets/tree/develop/updates>`__.
```
Comment (LefterisJP) [inline docs/contribute.rst:75]: Please try to use 1, max 2 words for a link

```suggestion
To add new assets for rotki you will have to edit the `SQL file<https://github.com/rotki/assets/tree/develop/updates>`__  in the last update at the assets repository.
```

File: docs/contribute.rst [hunk 2]
```diff
@@ -72,44 +72,31 @@ Our releases work like this:
...
+SQL sentences for insertion differ depending if we are adding an ethereum token or other types of assets. More information about each type of asset and
```
Comment (LefterisJP) [inline docs/contribute.rst:76]:
```suggestion
SQL sentences for insertion differ depending on if we are adding an ethereum token or other types of assets. More information about each type of asset and
```

File: docs/contribute.rst [hunk 3]
```diff
@@ -72,44 +72,31 @@ Our releases work like this:
...
+
+In order to do so you will need to install the dependencies in the `requirements.txt` file.
 
 .. _get_coingecko_asset_identifier:
 
 Get CoinGecko asset identifier
 --------------------------------
 
 In most cases the CoinGecko asset identifier matches the URL one, for example "weth" for `WETH <https://www.coingecko.com/en/coins/weth>`__. However, sometimes it doesn't, for example "sharering" for `SHR <https://www.coingecko.com/en/coins/sharetoken>`__ ("sharetoken" in the URL).
+Lately coingecko added the API id of the asset to the information provided for the asset.
```
Comment (LefterisJP) [inline docs/contribute.rst:99]: Can you take a screenshot of this and show to the user where it is? Like this.

**Review by LefterisJP — APPROVED** [review]
Thank you ser. :+1:

**Pull request merged**